charts/layer-zero/README.md

```diff
@@ -292,11 +292,25 @@ the `layer-zero-chart` and their default values.
 
 ### Service parameters
 
-| Name                  | Description                                                            | Value       |
-| --------------------- | ---------------------------------------------------------------------- | ----------- |
-| `service.type`        | Kubernetes service type, use "LoadBalancer" for external communication | `ClusterIP` |
-| `service.port`        | `SERVER_PORT` Service port                                             | `8081`      |
-| `service.annotations` | Annotations for the service                                            | `{}`        |
+| Name                         | Description                                                                                                      | Value       |
+| ---------------------------- | ---------------------------------------------------------------------------------------------------------------- | ----------- |
+| `service.type`               | Kubernetes service type, use "LoadBalancer" for external communication                                           | `ClusterIP` |
+| `service.port`               | `SERVER_PORT` Service port                                                                                       | `8081`      |
+| `service.annotations`        | Annotations for the service                                                                                      | `{}`        |
+| `service.keepResourcePolicy` | If type is "LoadBalancer" and this is set to true, the service will not be deleted when the chart is uninstalled | `false`     |
+
+### Ingress parameters
+
+| Name                       | Description                                             | Value                    |
+| -------------------------- | ------------------------------------------------------- | ------------------------ |
+| `ingress.enabled`          | Enable ingress controller resource                      | `false`                  |
+| `ingress.annotations`      | Annotations for the Ingress resource                    | `{}`                     |
+| `ingress.ingressClassName` | Ingress class name                                      | `nginx`                  |
+| `ingress.pathType`         | Ingress path type                                       | `ImplementationSpecific` |
+| `ingress.hosts`            | .host Hostname to your service                          | `[]`                     |
```

Comment:
Shouldn't it be `ingress.hosts.host` ? I think the .host is better suited for the name column.